plugins/ua/_universalarchive

```diff
@@ -0,0 +1,7 @@
+#compdef universalarchive
+#autoload
+
+_arguments \
+  '(-r --remove)'{-r,--remove}'[Remove archive.]' \
```

Comment:
You specify a `-r`/`--remove` option but there's none implemented. I think this is a copy-paste artifact from the `extract` plugin.